data-prepper-plugins/common/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
     api project(':data-prepper-api')
     implementation 'com.fasterxml.jackson.core:jackson-databind'
     implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
+    implementation 'com.github.ben-manes.caffeine:caffeine:3.1.8' 
```

Comment:
Is this available in our settings.gradle file? If not, we should move it there. If so, use that version.